tests/copy_cut_paste.cpp

```diff
@@ -28,36 +27,32 @@
 using namespace testing;
 using namespace wlcs;
 
-using CopyCutPaste = wlcs::InProcessServer;
-
 namespace
 {
+struct StartedInProcessServer : InProcessServer
```

Comment:
What is the need for this class? By keeping the server started between test cases, does it not risk one test corrupting another?